### DISCUSSION TITLE
Add additional tags to all vector  mertics

### DISCUSSION THREAD

**Discussion by LinTechSo** [vectordotdev/vector#12003] [General]
Hi, how can I add specific tags to all components metrics in the vector pipeline? is there any way?
in the transform section, I add some tags to my metrics but I want to add these specific tags to all vector metrics

**Comment by jszwedko** [comment]
Hi @LinTechSo ,

You can use the `remap` transform. See this example: https://vector.dev/docs/reference/configuration/transforms/remap/#examples-modify-metric-tags

You can place it after the `internal_metrics` source in your topology.
